glide.lock

```diff
@@ -1095,7 +1095,7 @@ imports:
 - name: github.com/pborman/uuid
   version: ca53cad383cad2479bbba7f7a1a05797ec1386e4
 - name: github.com/pelletier/go-toml
-  version: 16398bac157da96aa88f98a2df640c7f32af1da2
+  version: c01d1270ff3e442a8a57cddc1c92dc1138598194
```

Comment:
@deads2k unclear where the go-toml bump came from, but it does have updates we want like golang 1.10 support.

Reply:
> @deads2k unclear where the go-toml bump came from, but it does have updates we want like golang 1.10 support.

ok.  I'm not too worried about this one.  Just be sure that you never update `k8s.io/*` repos without asking @openshift/sig-master   That usually indicates a publishing bot failure.